sympy/parsing/c/c_parser.py

```diff
@@ -227,6 +252,15 @@ def transform_var_decl(self, node):
             types
```

Comment:
Can you also update the docstring with the supported data types?

Reply:
Sure... In the next commit!